hack/generators/conversion-webhook/main.go

```diff
@@ -92,7 +147,7 @@ const (
         service:
           name: %s
           namespace: %s
-          path: /convert
+          path: /convert%s
```

Comment:
Is this templating actually used anywhere?

Reply:
It's used here: https://github.com/Kong/kong-operator/blob/a2d25c389e06f49c09eeccb627cc4d3f9ea2a4c6/hack/generators/conversion-webhook/main.go#L226-L229